minidump/src/minidump.rs

```diff
@@ -2626,6 +2630,51 @@ impl MinidumpSystemInfo {
     pub fn cpu_info(&self) -> Option<Cow<str>> {
         self.cpu_info.as_deref().map(Cow::Borrowed)
     }
+
+    /// Strings identifying the version and build number of the operating
+    /// system.
+    ///
+    /// Tries to parse the version number from the build if it cannot be found
+    /// in the version string. If the stream already contains a valid version
+    /// number or parsing from the build string fails, this will return what's
+    /// directly stored in the stream.
+    pub fn os_parts(&self) -> (String, Option<String>) {
```

Comment:
I'm not entirely happy with this name, but I couldn't come up with a better alternative so I went with the name used by symbolic. Suggestions are very welcome here.